apps/editorial_features/components/eoy/client.coffee

```diff
@@ -163,17 +163,6 @@ module.exports.EoyView = class EoyView extends Backbone.View
   setupCarousel: ->
     initCarousel $('.carousel'), imagesLoaded: true
 
-  setImage: ->
-    $img = $(this)
-    src = $img.attr 'src'
-    if src != ''
-      width = 800
-      src = resize(src, width: width * (window.devicePixelRatio or 1))
-      $img.attr 'src', src
-
-  setImages: ->
```

Comment:
Instead of going through each image and replacing the src with a resized version, could we just give the template a `resize` function and have the template do the work in one sweep? Looked fine when I tested it but let me know if there was some other reason we had to do this.